vaadin-combo-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/combobox/test/LazyLoadingIT.java

```diff
@@ -85,16 +85,17 @@ public void scrollOverlay_morePagesLoaded() {
         assertLoadedItemsCount(
                 "There should be 150 items after loading three pages", 150,
                 stringBox);
-        assertRendered("Item 115");
+        assertRendered("Item 102");
     }
 
     @Test
     public void openPopup_scrollToEnd_onlyFirstAndLastPagesLoaded() {
         stringBox.openPopup();
         scrollToItem(stringBox, 1000);
+        waitUntil(e -> getOverlayContents().contains("Item 999"));
         assertLoadedItemsCount(
-                "Expected the first and the last pages to be loaded (100 items).",
-                100, stringBox);
+                "Expected the last page to be loaded (50 items).",
```

Comment:
The first page is discarded with the new version so there's only 50 items (the last page)